docs/lang/articles/contribution/dev_install.md

```diff
@@ -126,6 +125,12 @@ installer.
 3. Execute `python3 -m taichi test` to run all the tests. It may take
   up to 5 minutes to run all tests.
 
+## Conda
+
+To avoid directly installing Taichi's dependencies into your existing
+Python environment, we have provided a pre-defined `conda` environment.
+You can find the instructions [here](https://github.com/taichi-dev/taichi/conda/README.md).
+
```

Comment:
It also worth mentioning that once conda is set up following the README, they still need to run `python setup.py develop`?

Reply:
Done